scripts/poweron/power-on.py

```diff
@@ -40,30 +40,34 @@ def main(session, args):
     mode = session.xenapi.host.get_power_on_mode(remote_host)
```

Comment:
I'd prefer if we used an Enum to represent the values for mode, but that its only supported starting in python 3.4 or through a library, without talking about increased burden to maintain the values of the enum. :(